evasquez26/angular4-startup
# S2-ONE-132-133-124 Admin request list, role request details, accept/reject request

## Story / Task

see [ONE-132](https://bixlabs.atlassian.net/browse/ONE-132) [ONE-133](https://bixlabs.atlassian.net/browse/ONE-133) [ONE-124](https://bixlabs.atlassian.net/browse/ONE-124)


## What this PR does?

Display the list of role request, ability to see the role request details, and ability to accept/reject a request

### How to test / reproduce

- login to the app
- go to Admin (at the moment it assumes you're admin, TODO: service to eval role)
- You should see a list of role requests
- If you click details, you should see detail information about the request
- (In progress) accept/reject a request

## Migrations

NO


Thanks
